lib/suse/connect/cli.rb

```diff
@@ -149,6 +149,10 @@ def extract_options # rubocop:disable MethodLength
           @options[:write_config] = true
         end
 
+        @opts.on('--cleanup', 'remove old system credentials and all zypper services installed by SUSEConnect') do |opt|
```

Comment:
@wstephenson This is a command line option for the users. I don't think they even know what zypp is. But zypper command line tool is well known to everyone.